src/rust/engine/fs/store/src/snapshot.rs

```diff
@@ -57,7 +57,7 @@ impl Snapshot {
     S: StoreFileByDigest<Error> + Sized + Clone + Send + 'static,
     Error: fmt::Debug + 'static + Send,
   >(
-    store: Store,
+    _store: Store,
```

Comment:
Should this be removed in a followup?

Reply:
Hm, yes! I thought that I had, but that must have been in a separate commit.